<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDonor</class>
 <widget class="QMainWindow" name="AddDonor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>711</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:#dc143c;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_addDonor">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>331</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	font: 63 28pt &quot;Segoe UI Semibold&quot;;
	color: #353839;
}</string>
     </property>
     <property name="text">
      <string>NEW DONOR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_addDonor">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>280</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
background:#333;
color:white;
border-radius:20px;
}</string>
     </property>
     <property name="text">
      <string>Add Donor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_back">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
background:#333;
color:white;
border-radius:15px;
}</string>
     </property>
     <property name="text">
      <string>Back to Menu</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>301</width>
       <height>94</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">*{
color:#353839;
}</string>
          </property>
          <property name="text">
           <string>First Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_firstName">
          <property name="styleSheet">
           <string notr="true">*{
background:white;
color:black;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">*{
color:#353839;
}</string>
          </property>
          <property name="text">
           <string>Last Name: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_lastName">
          <property name="styleSheet">
           <string notr="true">*{
background:white;
color:black;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">*{
color:#353839;
}</string>
          </property>
          <property name="text">
           <string> Email:         </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_email">
          <property name="styleSheet">
           <string notr="true">*{
background:white;
color:black;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>150</y>
       <width>291</width>
       <height>94</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">*{
color:#353839;
}</string>
          </property>
          <property name="text">
           <string>Contact:      </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_contact">
          <property name="styleSheet">
           <string notr="true">*{
background:white;
color:black;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">*{
color:#353839;
}</string>
          </property>
          <property name="text">
           <string>Gender:       </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_gender">
          <property name="styleSheet">
           <string notr="true">*{
background:white;
color:black;
}</string>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Female</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Male</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Other</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">*{
color:#353839;
}</string>
          </property>
          <property name="text">
           <string>Blood Group</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_bloodGroup">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">*{
background:white;
color:black;
}</string>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>O+</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>O-</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>A+</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>A-</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B+</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B-</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AB+</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AB-</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
